WordPress/WordPressTest/Classes/Stores/StatsWidgetsStoreTests.swift

```diff
@@ -3,6 +3,7 @@ import JetpackStatsWidgetsCore
 
 @testable import WordPress
 
+// TODO: rewrite this system to avoid using BuildSettings.current (hardcoded all over the place)
```

Comment:
I tried to update it, but it required too many changes. I'm not sure it's worth considering the relatively low value of this test.